templates/registry/markdown/enum_macros.j2

```diff
@@ -0,0 +1,10 @@
+{% import 'stability.j2' as stability %}
+{% macro table(enum, notes) %}
+`{{enum.name}}` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+{% for espec in enum.type.members %}| `{{ espec.value }}` | {{ (espec.brief or espec.id) | trim }}{{ notes.add(espec.note) }} | {{ stability.badge(espec.stability, espec.deprecated) }} |
```

Comment:
taking into account that we always now generate id if brief is not there, should we skip to add brief if it's identical to the enum ID and has no specific description?

Reply:
This could be a follow on, but yes - I'm not finding brief to be very descriptive/useful in a lot of scenarios.